lib/rom/http/commands/create.rb

```diff
@@ -0,0 +1,17 @@
+module ROM
+  module HTTP
+    module Commands
+      class Create < ROM::Commands::Create
+        adapter :http
+
+        def execute(tuples)
+          Array([tuples]).flat_map do |tuple|
+            attributes = input[tuple]
+            validator.call(attributes)
+            relation.insert(attributes.to_h)
+          end.to_a
```

Comment:
This returns the relation which has not triggered an http request. What do you think about changing this to `end.map(&:to_a)`? (which triggers it for each tuple). Or is this handled somewhere else?